src/tests/ftest/pool/management_race.py

```diff
@@ -0,0 +1,134 @@
+#!/usr/bin/python3
+"""
+  (C) Copyright 2022 Intel Corporation.
+
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+"""
+
+import time
+import random
+from apricot import TestWithServers
+from thread_manager import ThreadManager
+from command_utils_base import CommandFailure
+
+class PoolManagementRace(TestWithServers):
+    """
+    Epic: Create system level tests that cover pool management race boundary tests and
+          functionality.
+    Testcase:
+          DAOS-8463: Pool management integrity with race boundary conditions
+    Test Class Description:
+          Start DAOS servers, create pools and delete the pool while query and list.
+    :avocado: recursive
+    """
+
+    def del_recreate_query_and_list_pools(
+            self, del_recreate=True, pool_num=1, thread_num=1, test_loop=2):
+        """ To delete and recreate pool with the same label, to query and list pools.
+        Args:
+            del_recreate (bool): flag to delete and recreate pool with the same label.
+            pool_num (int): pool number.
+            thread_num (int): thread number.
+            test_loop (int): number of test loop to be executed.
+        """
+
+        wait_time = self.params.get("wait_time", '/run/boundary_test/*')
+        max_query_time = self.params.get("max_query_time", '/run/boundary_test/*')
+        pool_label_prefix = self.params.get("pool_label_prefix", '/run/boundary_test/*')
+        label = pool_label_prefix + str(pool_num)
+        for test_num in range(test_loop):
+            if del_recreate:
+                destroy_result = self.get_dmg_command().pool_destroy(
+                    pool=self.pool[pool_num].label.value, force=1)
+                self.log.info(
+                    "--(%d.2.%d.%d)Pool %s deleted.\n",
+                    thread_num, pool_num, test_num, label)
+                new_pool = self.get_pool()
+                new_pool.set_property(prop_name="label", prop_value=label)
+                new_pool.label.update(label)
+                self.pool[pool_num] = new_pool
+                self.log.info(
+                    "--(%d.3.%d.%d)Pool %s recreated.\n",
+                    thread_num, pool_num, test_num, label)
+                # pool stays with a random time before destroy
+                pool_stay_time = random.randint(2, 5) #nosec
+                time.sleep(pool_stay_time)
+            else:
+                completed = False
+                start = float(time.time())
+                while not completed:
+                    try:
+                        self.log.info("--(%d.4.%d.%d)--pool querying.. \n",
+                                      thread_num, pool_num, test_num)
+                        query_result = self.get_dmg_command().pool_query(pool=label)
```

Comment:
```suggestion
                        self.get_dmg_command().pool_query(pool=label)
```
You don't need this variable if you won't use it.

Reply:
displace result was for the debug.